dozer/cogs/roles.py

```diff
@@ -292,6 +355,200 @@ async def rolecolor(self, ctx, role: discord.Role, color: discord.Color = None):
     `{prefix}rolecolor "Simbot Red"` - displays the color of the "Simbot Red" role in hexcode format
     `{prefix}rolecolor Bolb #FCC21B` - set the color of the "Bolb" role to #FCC21B
     """
+    async def update_role_menu(self, ctx, menu):
+        """Updates a reaction role menu"""
+        menu_message = await self.safe_message_fetch(ctx, menu=menu)
+
+        menu_embed = discord.Embed(title=f"Role Menu: {menu.name}")
+        menu_entries = await ReactionRole.get_by(message_id=menu.message_id)
+        for entry in menu_entries:
+            role = ctx.guild.get_role(entry.role_id)
+            menu_embed.add_field(name=f"Role: {role}", value=f"{entry.reaction}: {role.mention}", inline=False)
+        menu_embed.set_footer(text=f"React to get a role\nMenu ID: {menu_message.id}, Total roles: {len(menu_entries)}")
+        await menu_message.edit(embed=menu_embed)
+
+    @group(invoke_without_command=True, aliases=["reactionrole", "reactionroles"])
+    @bot_has_permissions(manage_roles=True, embed_links=True)
+    @has_permissions(manage_roles=True)
+    @guild_only()
+    async def rolemenu(self, ctx):
+        """Base command for setting up and tracking reaction roles"""
+        rolemenus = await RoleMenu.get_by(guild_id=ctx.guild.id)
+        embed = discord.Embed(title="Reaction Role Messages", color=blurple)
+        boundroles = []
+        for rolemenu in rolemenus:
+            menu_entries = await ReactionRole.get_by(message_id=rolemenu.message_id)
+            for role in menu_entries:
+                boundroles.append(role.message_id)
+            link = f"https://discordapp.com/channels/{rolemenu.guild_id}/{rolemenu.channel_id}/{rolemenu.message_id}"
+            embed.add_field(name=f"Menu: {rolemenu.name}", value=f"[Contains {len(menu_entries)} role watchers]({link})", inline=False)
+        unbound_reactions = await db.Pool.fetch(f"""SELECT * FROM {ReactionRole.__tablename__} WHERE message_id != all($1)"""
```

Comment:
I know there should be a way to port this to the ORM, but I don't know how to be honest....